src/utils/todont.ts

```
@@ -82,13 +78,103 @@ export const replaceText = ({
-const initializeTodont = () => {
+const ARCHIVE_COMMAND_LABEL = "TODONT Hotkey";
+
+const initializeTodont = (extensionAPI: OnloadArgs["extensionAPI"]) => {
   const unloads = new Set<() => void>();
-  return async (todontMode: typeof TODONT_MODES[number]) => {
-    if (todontMode !== "off") {
-      const TODONT_CLASSNAME = "roamjs-todont";
-      const css = document.createElement("style");
-      css.textContent = `.bp3-button.bp3-small.${TODONT_CLASSNAME} {
+  const cleanup = () => {
+    unloads.forEach((u) => u());
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)